### DISCUSSION TITLE
TF2CustAttr_Getters

### DISCUSSION THREAD

**Discussion by DosMike** [nosoop/SM-TFCustAttr#21] [Q&A]
Do the TF2CustAttr_GetX functions work like attribute hooks / TF2Attrib_HookValueX, or rather: Do i have to poll a value from loadout items or is it enough to poll from the client?

**Comment by nosoop** [comment]
They're effectively just a key / value store on the entity; no collation is performed.  You'll have to call the getter on each item and the client and perform the summation / product yourself.
